setup.cfg

```diff
@@ -79,6 +79,7 @@ testing =
     %(image)s
     matplotlib
     toolz
+    ipywidgets
```

Comment:
maybe additional extras for `ipywidgets` to install with backend by one command?